### ISSUE TITLE
[v2] Better error messages

### ISSUE THREAD

**Issue by akaufmann** [imba/imba#274]
### Details

If by accident or because you don't know better, an HTML tag like `<br/>` is added, you get unhelpful error messages.

### Expected Behavior

An error message that indicates exactly what and where it is wrong. 
Elm-like error messages preferred. ;) 

### Actual Behavior
Error message like...
```js
bundling error {
  code: 'ERROR',
  error: [ImbaParseError: Parse error at [xxx:1]: Unexpected 'MATH'] {
    error: { message: "Parse error at [xxx:1]: Unexpected 'MATH'" },
    message: "Parse error at [xxx:1]: Unexpected 'MATH'",
    filename: undefined,
    line: undefined,
    _options: { pos: xxx, tokens: [Array], token: [Token], meta: [Object] },
    severity: 'error',
    _code: 'tag app-foo\n' +
      '  def render\n' +
      '    <self>\n' +
      '          <div>\n' +
      '                "Lorem ipsum"\n' +
      '                <br/>\n' +
      '                ...'
```
if you add a `<br/>`

**Comment by somebee** [comment]
This is the error printed by imba-rollup? Should definitely be better. We should probably support self-closing `/>` as well, don't you think?

**Comment by akaufmann** [comment]
> This is the error printed by imba-rollup? 

Yes - I thought the error message would be passed 1:1 to Rollup. I checked it with `imbac` and here is the error message:

```js
Parse error at [1203:1]: Unexpected 'MATH'
      4 |           <div>
      5 |             "Lorem ipsum"
   -> 6 |             <br/>
      7 |             ...
```
This is better! Only the 'MATH' doesn't tell me anything. I suspect lmba is assuming that the slash is a Division operator.

> We should probably support self-closing /> as well, don't you think?

That would be great! This would make copying and pasting existing HTML a bit easier and since you don't have a closing tag in Imba like LitElement or a self-closing component like in React or Vue, it would be nicer to declare self-closing tags/components this way - at first it was a bit unfamiliar for me not to add a self-/closing tag.

**Comment by somebee** [comment]
The latest version of imba-rollup should show better error messages. In dev mode it will also compile the bundle and print the error message in the bundle, so you can see it in the browser with live-reloading etc. Looks pretty basic, but a good start :)
